hassio/src/dashboard/hassio-addons.ts

```diff
@@ -77,19 +79,23 @@ class HassioAddons extends LitElement {
   }
 
   private _computeIconTitle(addon: HassioAddonInfo): string {
+    if (addon.state !== "started") {
+      return "Add-on is stopped";
+    }
     if (addon.installed !== addon.version) {
       return "New version available";
     }
-    return addon.state === "started"
-      ? "Add-on is running"
-      : "Add-on is stopped";
+    return "Add-on is running";
   }
 
   private _computeIconClass(addon: HassioAddonInfo): string {
```

Comment:
I guess all these one off functions to fill properties are from the Polymer days. We could consider just ditching it and inlining everything.